testsuite/python/polymer.py

```diff
@@ -21,65 +21,185 @@
 import espressomd
 from espressomd.interactions import FeneBond
 from espressomd import polymer
+from espressomd.shapes import Wall
 
 
-class Polymer(ut.TestCase):
-    system = espressomd.System(box_l=[1.0, 1.0, 1.0])
+class PolymerPositions(ut.TestCase):
+    box_l = 15
+    seed = 23
+
+    system = espressomd.System(box_l=[box_l, box_l, box_l])
     np.random.seed(1234)
     system.set_random_state_PRNG()
 
-    num_mono = 5
+    def assertShape(self, positions, n_poly, n_mono):
+        """
+        Assert that positions array has expected shape and
+        expected number of elements.
+
+        """
+        shape = np.shape(positions)
+        self.assertEqual(shape, (n_poly, n_mono, 3))
+
+    def assertBondLength(self, positions, bond_length):
+        """
+        Assert that distance between monomers is bond_length.
+
+        """
+        for p in positions:
+            distances = np.linalg.norm(p[1:] - p[:-1], axis=1)
+            max_deviation = np.max(abs(distances - bond_length))
+            self.assertLess(max_deviation, 1e-10)
+
+    def assertBondAngle(self, positions, bond_angle, bond_length):
```

Comment:
please adhere to the pep8 naming convention for methods: lowercase with underscores to separate words

Reply:
I tried to be consistent with the other asserts from pythons module [unittest](https://docs.python.org/2/library/unittest.html#assert-methods), which are all camel case (`assertEqual`, `assertRaisesRegex()`). I can change the nomencalture here, but personally I am in favour of consistency amongst the asserts.